react/Portal/index.jsx

```diff
@@ -1,6 +1,12 @@
-import ReactDOM from 'react-dom'
-const Portal = ({ into, children }) => {
-  const targetElement = document.querySelector(into)
-  return ReactDOM.createPortal(children, targetElement)
+let Portal
+if (process.env.USE_REACT) {
```

Comment:
I know it is a breaking change and #594 should have said so. But, there is a very weird behavior here: we use React for everything by default, but we need to use a environment variable for Portal?